test/sanity/code-smell/no-tests-as-filters.py

```diff
@@ -44,6 +44,8 @@
     'skip': 'skipped',
 }
 
+IGNORED_TESTS = ('list')
```

Comment:
Future additions to this list will result in cleaner diffs if this is reformatted:

```python
IGNORED_TESTS = (
    'list',
)
```

Reply:
Also, why does this test need to be ignored?

Reply:
I'm guessing because jinja2 provides a `list` filter: http://jinja.pocoo.org/docs/dev/templates/#list

Reply:
Although I'm concerned about adding both a filter and test by the same name.

Reply:
Well, `sequence` is already taken and `tuple` doesn't really describe what it is. But if we consider `string`, it exists both as a filter as well as a test. That's why I could live with this.

I proposed this upstream, so we may want to delay this until they have made a decision ?